lib/sshutils/server.go

```diff
@@ -450,7 +453,7 @@ func (s *Server) HandleConnection(conn net.Conn) {
 				connClosed()
 				return
 			}
-			go s.newChanHandler.HandleNewChan(wconn, sconn, nch)
+			go s.newChanHandler.HandleNewChan(ccx, wconn, sconn, nch)
```

Comment:
In the comments for `ConnectionContext` you mention it's meant to be a wrapper around `ssh.ServerConn`, any reason you didn't include `sconn` (and `wconn`) into the `ConnectionContext`?